pkgs/development/libraries/geoclue/default.nix

```diff
@@ -1,24 +1,29 @@
 { fetchurl, stdenv, intltool, pkgconfig, gtk-doc, docbook_xsl, docbook_xml_dtd_412, glib, json-glib, libsoup, libnotify, gdk_pixbuf
 , modemmanager, avahi, glib-networking, wrapGAppsHook, gobjectIntrospection
-, withDemoAgent ? false
+# doc build currently seems to be broken
```

Comment:
Could you provide more details? The docs build for me.

Reply:
If I set this to true I get the following:
```
> nix build -f . pkgs.geoclue
builder for '/nix/store/mgyaj5vnzg1fy2yvx16xqk11r6kiqq9v-geoclue-2.4.11.drv' failed with exit code 2; last 10 log lines:
  ../libgeoclue-docs.xml:77: element include: XInclude error : could not load ../xml/gclue-client-proxy.xml, and no fallback was found
  warning: failed to load external entity "../xml/gclue-location-proxy.xml"
  ../libgeoclue-docs.xml:83: element include: XInclude error : could not load ../xml/gclue-location-proxy.xml, and no fallback was found
  make[3]: *** [Makefile:750: html-build.stamp] Error 6
  make[3]: Leaving directory '/build/geoclue-2.4.11/docs/lib'
  make[2]: *** [Makefile:577: all-recursive] Error 1
  make[2]: Leaving directory '/build/geoclue-2.4.11/docs'
  make[1]: *** [Makefile:517: all-recursive] Error 1
  make[1]: Leaving directory '/build/geoclue-2.4.11'
  make: *** [Makefile:449: all] Error 2
[0 built (1 failed), 0.0 MiB DL]
error: build of '/nix/store/mgyaj5vnzg1fy2yvx16xqk11r6kiqq9v-geoclue-2.4.11.drv' failed
```

This is with sandboxing on. I don't get this in the previous version, but I couldn't figure out what the problem.

Reply:
Recent master seems to have a bunch of build-related commits, so I suspect they may still be fixing things up.